tests/plugins/test_bump_release.py

```diff
@@ -180,11 +180,8 @@ def ssl_login(self, cert=None, ca=None, serverca=None, proxyuser=None):
 
         parser = df_parser(plugin.workflow.builder.df_path, workflow=plugin.workflow)
         assert parser.labels['release'] == next_release['expected']
-        # Old-style spellings will be asserted only if other old-style labels are present
-        if 'BZComponent' not in parser.labels.keys():
-            assert 'Release' not in parser.labels
-        else:
-            assert parser.labels['Release'] == next_release['expected']
+        # Old-style spellings should not be asserted
+        assert 'Release' not in parser.labels
```

Comment:
Are we sure this change is correct? I'm not sure why it is needed. The test is slightly odd because (unless I'm reading it wrong) it doesn't set a release/Release label to begin with.

Reply:
self.prepare returns a BumpReleasePlugin
plugin.run calls the run method of that plugin.
pre_bump_release.py::run() line 147 sets the release_label to next_release

Reply:
I see where it sets the next_release, i.e. the one we want to end up with. I don't see where actual_release, i.e. the one we started with in the Dockerfile, is set. @rcerven?